cypress/integration/jsGrowthTableDOM.spec.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+Cypress.Commands.add('multiClick',
+(selector) => {
+  for (let n = 0; n < 7; n++) {
+    cy.get(selector).click( {force: true} );
+  }
+});
+
+describe('Table', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.get('tbody > tr').as('rows');
+    cy.get('tbody :nth-child(1) :nth-child(n)').as('columns');
+    cy.get('.remove-row').click().as('rmRow');
+    cy.get('.remove-column').click().as('rmColumn');
+  });
+
+  it('should have 4 rows by default', () => {
+    cy.get('@rows').should('have.length', 4)
+  });
+
+  it('should have 4 columns by default', () => {
+    cy.get('@columns').should('have.length', 4)
+  });
+
+  it.only('should have max 10 columns', () => {
```

Comment:
what does it.only mean and why do we have it here?

Reply:
forgot to remove that